src/models/hooks/sessionReportPilot.js

```diff
@@ -213,6 +213,76 @@ export const createGoalsForSessionRecipientsIfNecessary = async (sequelize, sess
   }
 };
 
+export const removeGoalsForSessionRecipientsIfNecessary = async (sequelize, sessionReportOrInstance, options) => {
+  const processSessionReport = async (sessionReport) => {
+    let event;
+    let nextSessionRecipients;
+
+    if (sessionReport && sessionReport.data) {
+      const data = (typeof sessionReport.data.val === 'string')
+        ? JSON.parse(sessionReport.data.val)
+        : sessionReport.data;
+
+      event = data.event;
+      nextSessionRecipients = data.recipients;
```

Comment:
Why call it 'next' session recipients?

Reply:
Because it is the new value of recipients being saved.

Reply:
I'm open to suggestions for variable names if you think it needs to be more clear